con-base/src/main/java/ba/codecentric/base/domain/Hotel.java

```diff
@@ -62,6 +57,14 @@
     @Size(max = 500, message = "Description is too long, 500 characters allowed")
     private String description;
 
+    @NotNull(message = "Please add Check-in time")
+    @Column(name = "check_in")
+    private Time checkIn;
```

Comment:
For such simple task we don't need to have such overdo. Time data type is just enough. I would like to hear opinion for others, but I think in this case we don't have any need for LocalTime. With Time we can also handle seconds in setters and have it without it on frontend. String wasn't an option, but implementation with LocalTIme and getters which returns String, was exactly same thing. So, my opinion is to stay with Time data type.